tasks/deps-debian.yml

```diff
@@ -16,4 +16,11 @@
     state: link
   when: ansible_distribution_release == 'bionic'
 
+# see https://github.com/TACC/Lmod/commit/1d15717144f595b9a09e8421aa00b5dead3ed21d
+- name: fix lua-posix on Ubuntu 20.04
+  file:
+    src: /usr/lib/x86_64-linux-gnu/liblua5.2-posix.so
+    dest: /usr/lib/x86_64-linux-gnu/lua/5.2/posix.so
+    state: link
+  when: ansible_distribution_release == 'focal'
```

Comment:
Please also check that OS/Distro matches Ubuntu.

Reply:
sorry for the late reply, I am unsure what you would like here . This is just an update on the section above for bionic, which does not seem to do this.

Reply:
Sorry, I should have been clearer: please also check that `ansible_distribution == 'Ubuntu'`, thus: 

```yml
  when: >-
    ansible_distribution == 'Ubuntu'
    and
    ansible_distribution_release == 'focal'
```

It's a bit safer by checking for both.

Reply:
Hi I have made the requested changes - are they as you expect?